cloudformation/master-server-substack.cfn.yaml

```diff
@@ -24,6 +24,8 @@ Parameters:
     Type: String
   MasterENI:
     Type: String
+  MasterPublicIp:
```

Comment:
Similar to previous comment, we can rename this to `UseMasterPublicIpInSubstack`

Reply:
Renamed to UseMasterPublicIp